tests/rule_based_profiler/parameter_builder/test_simple_date_format_string_parameter_builder.py

```diff
@@ -79,9 +87,8 @@ def test_simple_date_format_parameter_builder_instantiation():
         )
     )
 
-    assert date_format_string_parameter.CANDIDATE_STRINGS == DEFAULT_CANDIDATE_STRINGS
     assert date_format_string_parameter.threshold == 1.0
-    assert date_format_string_parameter.candidate_strings is None
+    assert date_format_string_parameter.candidate_strings == DEFAULT_CANDIDATE_STRINGS
```

Comment:
`CANDIDATE_STRINGS` is no longer a class level attribute. We are now checking that `DEFAULT_CANDIDATE_STRINGS` was set at instantiation if `candidate_strings` was not passed.